imx/interface.go

```diff
@@ -24,6 +24,13 @@ var (
 		CoreContractAddress:         "0x5FDCCA53617f4d2b9134B29090C87D01058e27e9",
 		ChainID:                     big.NewInt(1),
 	}
+	Ropsten = Environment{
+		BaseAPIPath:                 "https://api.ropsten.x.immutable.com",
+		EthereumRPC:                 "https://eth-ropsten.alchemyapi.io/v2/",
```

Comment:
Ropsten is deprecated. https://eth-ropsten.alchemyapi.io/ now returns 404 so it is pointless to have this config.